go/vt/vtgate/executor.go

```diff
@@ -283,25 +292,50 @@ func (e *Executor) handleExec(ctx context.Context, safeSession *SafeSession, sql
 	return qr, err
 }
 
+func (e *Executor) handleRangeExec(ctx context.Context, safeSession *SafeSession, sql string, bindVars map[string]*querypb.BindVariable, target querypb.Target, keyRange *topodatapb.KeyRange, logStats *LogStats) (*sqltypes.Result, error) {
```

Comment:
For consistency with `shardExec` I feel like this should be called `shardRangeExec`

Reply:
Done.